BE/toDoListAPI/src/main/java/shcool/mindera/toDoListAPI/controller/UserController.java

```diff
@@ -1,8 +1,10 @@
 package shcool.mindera.toDoListAPI.controller;
-import org.springframework.web.bind.annotation.RequestMapping;
-import org.springframework.web.bind.annotation.RestController;
+import org.springframework.web.bind.annotation.*;
+import shcool.mindera.toDoListAPI.model.*;
 import shcool.mindera.toDoListAPI.service.UserService;
 
+import java.util.List;
```

Comment:
Sei que e chato passei por isso tambem, mas cuidado com os espacos
```suggestion
import org.springframework.web.bind.annotation.*;
import shcool.mindera.toDoListAPI.model.*;
import shcool.mindera.toDoListAPI.service.UserService;
import java.util.List;
```